AlembicImporter/Packages/com.unity.formats.alembic/Runtime/Scripts/Misc/PinnedArray.cs

```diff
@@ -103,6 +103,10 @@ System.Collections.IEnumerator System.Collections.IEnumerable.GetEnumerator()
     }
 
 
+}
+
+namespace UnityEngine.Formats.Alembic.Sdk
+{
```

Comment:
The PinnedList class is also used in the Sdk namespace, to marshall data to/from c++

Reply:
should PinnedObject also be in the sdk namespace?

Reply:
I don't know. Only PinnedList is used by the Sdk. The other ones (PinnedObject and PinnedArray) are used by higher-level assemblies. 

But I find it would be more elegant to have them all in the same namespace so I will move them